static/src/javascripts/projects/commercial/modules/dfp/dfp-env.js

```diff
@@ -63,4 +63,4 @@ const dfpEnv: DfpEnv = {
     },
 };
 
-export default dfpEnv;
+export { dfpEnv };
```

Comment:
Let's go all the way and remove `dfpEnv` entirely, instead export the object literal.